grpc-api/server/native/README.md

```diff
@@ -8,11 +8,5 @@
 
 produces
 
-* `grpc-api/server/native/build/distributions/grpc-api-native-<version>.tar`
-* `grpc-api/server/native/build/distributions/grpc-api-native-<version>.zip`
-
-### Run
-
-```shell
-./gradlew grpc-api-server-native:run
```

Comment:
this is still runnable, no?

Reply:
:+1: Jianfeng and I have a short-ish set of steps to make py work from either netty or jetty run locally, without docker.